code/game/machinery/iv_drip.dm

```diff
@@ -316,7 +316,8 @@
 	var/datum/reagents/container = get_reagents()
 	log_combat(usr, target, "attached", src, "containing: ([container.get_reagent_log_string()])")
 	add_fingerprint(usr)
-	target.throw_alert(ALERT_IV_CONNECTED, /atom/movable/screen/alert/iv_connected)
+	if(isliving(target))
+		INVOKE_ASYNC(target, TYPE_PROC_REF(/mob/living, throw_alert), ALERT_IV_CONNECTED, /atom/movable/screen/alert/iv_connected)
```

Comment:
Is there a reason in particular why it's changed to async? Just wondering

Reply:
Can it be done as simply a TYPE_PROC_REF?

Reply:
Because I was dumb, fixed.